lib/Helper/Translate.php

```diff
@@ -120,6 +120,50 @@ public static function InitLocale($config, $language = NULL)
         self::$jsLocaleRequested = str_replace('_', '-', self::$requestedLanguage);
     }
 
+    /**
+     * Get translations for user selected language
+     * @param $config
+     * @param $language
+     * @return Translator|null
+     */
+    public static function getTranslationsFromLocale($config, $language): ?Translator
```

Comment:
Just to test this with you - this method is only ever going to be called with a provided `$language`, so why do we need the config object?

Reply:
Upon checking, this should not be here anymore as we agreed to have no default language set and `return null` when the language is not supported.